src/System.Numerics.Vectors/tests/Vector3Tests.cs

```diff
@@ -61,7 +61,6 @@ public void Vector3GetHashCodeTest()
             Assert.False(v4.GetHashCode() == v8.GetHashCode());
             Assert.False(v7.GetHashCode() == v6.GetHashCode());
             Assert.False(v8.GetHashCode() == v6.GetHashCode());
-            Assert.True(v8.GetHashCode() == v7.GetHashCode());
```

Comment:
The tests are looking for collisions (with a specific set of data, granted...).  The fact that part of that set no longer collides is noteworthy.